core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/SubmissionCostService.scala

```diff
@@ -6,7 +6,7 @@ import org.broadinstitute.dsde.rawls.model.UserInfo
 import org.broadinstitute.dsde.workbench.model.google.GoogleProject
 import org.broadinstitute.dsde.workbench.google.HttpGoogleBigQueryDAO
 
-import scala.collection.JavaConverters
+import scala.collection.JavaConverters._
```

Comment:
this change allows the use of `.asScala` instead of needing to call `JavaConverters.iterableAsScalaIterable`

Reply:
Nice!